scripts/nexus-space-usage.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+set -exo pipefail
+
+nexus-get-blob-usage(){
+    NEXUS_USERNAME="$(kubectl -n nexus get secret nexus-admin-credential --template '{{.data.username}}' | base64 -d)";
+    NEXUS_PASSWORD="$(kubectl -n nexus get secret nexus-admin-credential --template '{{.data.password}}' | base64 -d)";
+    BASIC_AUTH=$(echo -ne "${NEXUS_USERNAME}:${NEXUS_PASSWORD}" | base64 --wrap 0)
```

Comment:
I think we should replace the curl call with a tiny Python script, to avoid having to pass the credentials on the command line like with curl.

```suggestion
    export NEXUS_USERNAME NEXUS_PASSWORD
```